SQL/database_changelog.txt

```diff
@@ -1,17 +1,32 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 7.0; The query to update the schema revision table is:
+The latest database version is 7.1; The query to update the schema revision table is:
 
-INSERT INTO `schema_revision` (`major`, `minor`) VALUES (7, 0);
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (7, 1);
 or
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (7, 0);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (7, 1);
 
 In any query remember to add a prefix to the table names if you use one.
 
 -----------------------------------------------------
 
 -----------------------------------------------------
 
+Version 7.1, 9 September 2023, by that0neperson
+Added customization for quirks.
+
+ALTER TABLE `SS13_characters`
+	ADD COLUMN `quirk_prosthetic_limb_location` VARCHAR(64) COLLATE 'utf8mb4_general_ci',
```

Comment:
These should include `AFTER` clauses, so they are inserted at a sensible location order-wise in the table schema (as defined in beestation_schema.sql).

So 

```
ADD COLUMN `quirk_prosthetic_limb_location` VARCHAR(64) COLLATE 'utf8mb4_general_ci' AFTER `all_quirks`,
```

and so on

Reply:
added! ty for the help